src/main/java/com/example/gradecalculator/entities/GradeSystem.java

```diff
@@ -23,14 +23,18 @@ public class GradeSystem {
     @OneToMany(mappedBy = "gradeSystem")
     private List<SchoolYear> schoolYears;
 
-    public GradeSystem() {
+    public GradeSystem(String gradeSystem) {
```

Comment:
what does this constructor do? does it have any value?

Reply:
removed

Reply:
You mean this line: `gradeSystem = new GradeSystem("Grade System");` ?
But what does the String-Parameter actually do? The value is never used or assigned so it has no reason to exist.

Reply:
And FYI the implementation does dictate the test not otherwise.

Reply:
I tried to remove it but then testing didn't work and was showing errors

Reply:
Then change the test :)

Reply:
code changed

Reply:
There are still no new commits for this PR